index.html

```diff
@@ -169,10 +169,12 @@ <h2 class="color-secondary">About me</h2>
     <section class="contact">
         <div class="email">
             <h4 id="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h4>
-            <input class="color-primary" type="text" placeholder="Full name" required>
-            <input class="color-primary" type="email" placeholder="Email address" required>
-            <textarea placeholder="Enter text here" name="comment" cols="30" rows="10" required></textarea>
-            <button class="get-intouch-btn color-primary">Get in touch</button>
+            <form action="https://formspree.io/f/xpzewrow" method="post">
+                <input class="color-primary" type="text" placeholder="Full name" required maxlength="30">
+                <input class="color-primary" type="email" placeholder="Email address" required>
+                <textarea name="comment" cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
```

Comment:
After typing in each field, I realised that the entered data does not show because you didn't set the colour for these fields. This makes it difficult for users to edit a field or even double-check what they have entered. I suggest you add a colour for the inputs field to improve the UX.

See below

![input](https://user-images.githubusercontent.com/84930355/213251548-a026b919-fecf-4a93-821f-1f08423ecc7a.JPG)